src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 800;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
Do you really need shadow in general but not only for the focus state?
<img width="391" alt="image" src="https://user-images.githubusercontent.com/25587299/209983664-2cff3009-8503-44e2-a085-fd78e9348f4a.png">